JSONWriter.java

```diff
@@ -130,6 +129,35 @@ private JSONWriter append(String string) throws JSONException {
     }
 
     /**
+     * Append a value, converting it into a JSON string.
+     *

```

Comment:
Similar to append() method above, writes directly using JSONObject.writeValue() to encode the value.